backend/services/db.py

```
@@ -1,39 +1,87 @@
+
+        try:
+            _pool = await asyncpg.create_pool(
+                dsn=database_url,
+                min_size=1,
+                max_size=10,
+                command_timeout=10,
+            )
+            _last_status = "ok"
+            _last_error = ""
+            logger.info("✅ DB pool initialized")
+        except Exception as exc:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)